bindings/pyroot/pythonizations/src/PyROOTWrapper.cxx

```diff
@@ -59,9 +39,4 @@ void PyROOT::Init()
 
    // Memory management
    gROOT->GetListOfCleanups()->Add(&GetRegulatorCleanup());
-
-   // Bind ROOT globals that will be needed in ROOT.py
-   AddToGlobalScope("gROOT", gROOT, gROOT->IsA()->GetName());
-   AddToGlobalScope("gSystem", gSystem, gSystem->IsA()->GetName());
-   AddToGlobalScope("gInterpreter", gInterpreter, gInterpreter->IsA()->GetName());
```

Comment:
`gInterpreter` is also a macro, after this can you still use `gInterpreter` on the `Python` prompt?

Reply:
It does, even with vanilla cppyy! I guess there is some magic inside cppyy to make this work.